addons/portal_rating/controllers/portal_chatter.py

```diff
@@ -34,6 +34,13 @@ def portal_chatter_init(self, res_model, res_id, domain=False, limit=False, **kw
         result.update(self._portal_rating_stats(res_model, res_id, **kwargs))
         return result
 
+    def _get_void_portal_messages_filter(self):
+        """Override the method to include messages with ratings."""
+        res = super()._get_void_portal_messages_filter()
+        if request.env.context.get("rating_include"):
+            return lambda m: res(m) or m.rating_ids
+        return res
+
```

Comment:
I don't get what's the purpose of this method

Reply:
Seem to me like modifying the domain should be enough to show messages without body.